src/main/java/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -1,32 +1,123 @@
+        }
+    }
     public static class Component {
-        public static CustomXbox driverXbox;
+        public static PDP pdp;
+
+        public static TankDriveShifting chassis;
+		public static Motor leftWheel;
+		public static Motor rightWheel;
+        public static SolenoidShifters shifter;
+        public static EnableableModifier rightWheelAccelerationCap;
+        public static EnableableModifier leftWheelAccelerationCap;
+        
+		public static CANEncoder leftWheelEncoder;
+		public static CANEncoder rightWheelEncoder;
+		public static EncoderPair chassisEncoders;
+        public static CustomPIDController chassisTurnMC;
+		public static CustomPIDController drivePID;
+		public static NavX navx;
     }
     public static class HumanInput {
         public static class Driver {
             public static CustomXbox xbox;
         }
-        public static class Operator {}
+        public static class Operator {
+			public static CustomJoystick joystick;
+		}
     }
     public RobotMap() {
-        Component.driverXbox = new CustomXbox(Port.HumanInput.xboxController);
-		Component.driverXbox.setDeadZone(0.1);
+        Component.pdp = new PDP();
+
+        // Wheels
+        Component.leftWheelEncoder = new CANEncoder("LeftEncoder", Port.CAN.leftEncoder);
+        Component.rightWheelEncoder = new CANEncoder("RightEncoder", Port.CAN.rightEncoder);
+        Component.leftWheelEncoder.setDistancePerPulse(Metrics.Wheel.INCHES_PER_TICK);
+        Component.rightWheelEncoder.setDistancePerPulse(Metrics.Wheel.INCHES_PER_TICK);
+		Component.leftWheelAccelerationCap = new EnableableModifier(new AccelerationCap(Component.pdp));
+		Component.leftWheelAccelerationCap.enable();
+		Component.rightWheelAccelerationCap = new EnableableModifier(new AccelerationCap(Component.pdp));
+		Component.rightWheelAccelerationCap.enable();
+        Component.leftWheel = new Motor("LeftWheel", Component.leftWheelAccelerationCap,
+			new CANTalonSRX(Port.PWM.leftDriveA), new CANTalonSRX(Port.PWM.leftDriveB));
+		Component.rightWheel = new Motor("RightWheel", Component.rightWheelAccelerationCap,
+            new CANTalonSRX(Port.PWM.rightDriveA), new CANTalonSRX(Port.PWM.rightDriveB));
+		// Chassis
+		Component.shifter = new SolenoidShifters(Port.Pneumatics.shifter.pcmID, Port.Pneumatics.shifter.forward,
+            Port.Pneumatics.shifter.reverse);
+        Component.chassisEncoders = new EncoderPair(Component.leftWheelEncoder, Component.rightWheelEncoder);
+        Component.chassis = new TankDriveShifting("2019-Chassis", Component.leftWheel, Component.rightWheel, Component.shifter);
+        HumanInput.Driver.xbox = new CustomXbox(Port.HumanInput.xboxController);
+        HumanInput.Driver.xbox.setDeadZone(HumanInterfaceConfig.XBOX_DEADZONE);
+        HumanInput.Operator.joystick = new CustomJoystick(Port.HumanInput.joystick);
+        HumanInput.Operator.joystick.setDeadzone(HumanInterfaceConfig.STICK_LEFT_DEADZONE);
```

Comment:
the stick deadzone is set to`STICK_LEFT_DEADZONE`, and `STICK_RIGHT_DEADZONE` is left unused. What is the purpose of `STICK_RIGHT_DEADZONE` and where should it be used?